### DISCUSSION TITLE
Preview suggestions in rendered view

### DISCUSSION THREAD

**Discussion by drewvolz** [StoDevX/AAO-React-Native#5969] [Ideas]
For hours and dictionary report views we could provide a read-only view that applies the suggested props the user will send us in their report -- so a visual diff of the changes on the detail view they're changing.

As for the flow, it could be in addition to the submit button or an integrated step before we reveal the submit button.

**Comment by drewvolz** [comment]
Additionally the edit form could be used as a creation form since new buildings do get added.

**Comment by hawkrives** [comment]
https://github.com/StoDevX/AAO-React-Native/pull/7912 will do this for dictionary. If this works well, we can extend it to building hours.
